copy/single.go

```diff
@@ -149,6 +150,28 @@ func (c *copier) copySingleImage(ctx context.Context, unparsedImage *image.Unpar
 		ic.compressionFormat = c.options.DestinationCtx.CompressionFormat
 		ic.compressionLevel = c.options.DestinationCtx.CompressionLevel
 	}
+	// HACK: Don’t combine zstd:chunked and encryption.
+	// zstd:chunked can only usefully be consumed using range requests of parts of the layer, which would require the encryption
+	// to support decrypting arbitrary subsets of the stream. That’s plausible but not supported using the encryption API we have.
+	// Also, the chunked metadata is exposed in annotations unencrypted, which reveals the TOC digest = layer identity without
+	// encryption. (That can be determined from the unencrypted config anyway, but, still...)
+	//
+	// Ideally this should query a well-defined property of the compression algorithm (and $somehow determine the right fallback) instead of
+	// hard-coding zstd:chunked / zstd.
```

Comment:
Is there anything here that we should let the end user know in Podman/Bulidah/Skopeo?  Perhaps a warning in the compression options that zstd:chunked can't be encrypted?  No need to change for this PR, just a point to consider for later.

Reply:
For Skopeo, the user would have to ask for zstd:chunked on the CLI.

For Buildah/Podman, the default format is in `containers.conf`. For now I have filed https://github.com/containers/common/issues/2117, and I’ll clean that up at some point. Thanks for the suggestion!